Tests/iaas/flavor-naming/flavor_names.py

```diff
@@ -171,20 +174,29 @@ class Disk:
     disksize = OptIntAttr("#.GB Disk")
     disktype = TblAttr("Disk type", {'': '(unspecified)', "n": "Networked", "h": "Local HDD", "s": "SSD", "p": "HiPerf NVMe"})
 
+    def shorten(self):
+        return self
```

Comment:
We could drop disktypes that are networked here, but that would be new functionality... Good to have the option now, but it's a separate discussion, I guess.

Reply:
Disk type could even be replaced by `''` in any case (even if it's `n`)